.golangci.yml

```diff
@@ -0,0 +1,14 @@
+run:
+  deadline: 120s
+linters:
+  enable:
+    - govet
+    # TODO: Not supported by golang-ci - vetshadow
```

Comment:
> govet (vet, vetshadow)

According to the docs, looks like `vetshadow` is inside `govet`, so I think this TODO can be removed?

Reply:
Agreed it is. Fixed.